### ISSUE TITLE
Add the ability to hide some versions

### ISSUE THREAD

**Issue by MOZGIII** [pgsandstrom/package-json-upgrade#17]
Some things - like `@types/node` - are often not intended to be bumped to the next major version by design. Yet, the extension highlights them, which is annoying.

Can we add a configuration option to skip some packages or "package + version-spec" pairs?

I would set it at `.vscode/settings.json` something like this: 
```jsonc
{
  "packageJsonUpgrade.ignore": {
    "@types/node": "*" // I imagine this could be "14" to ignore the vesrion 14, or ">= 12" to ignore any major version after v12, but still allow version bumps to minor/patch versions
  }
}
```

And then, given my `package.json` has `@types/node` set to `^12.19` the extension won't suggest the upgrade, even though the latest version is 14.

**Comment by pgsandstrom** [comment]
That sounds like a great feature! I will try to find time to implement it within a few weeks.

**Comment by pgsandstrom** [comment]
I just released version 1.5.0 which adds this feature. The configuration is called "ignoreVersions" and works pretty much like you suggested. Just add a semver range that you want to ignore. If you want to ignore several semver ranges, you can put the ranges in an array like this:

```jsonc
{
  "packageJsonUpgrade.ignoreVersions": {
    "@types/node": ["1.2.0", ">=2"]
  }
}
```

Please let me know if you find any bugs or have any other feedback.
